### PR TITLE
Remove references to the 3rd party for helm chart values and docs

### PR DESCRIPTION
**Pull request by charlie-haley**
### Review

@reviewer

### Summary
Remove reference to 3rd party domain for helm chart values

### Reason
Security risk of fetching helm chart values from a third party domain, it's unnecessary, they should be created locally

### CODE REVIEW

File: app/kubernetes-ingress-controller/1.1.x/deployment/k4k8s-enterprise.md [hunk 1]
```diff
@@ -137,8 +137,23 @@ $ helm repo update
 # Helm 3
 $ helm install kong/kong --generate-name
     --namespace kong \
-    --values https://l.yolo42.com/k4k8s-enterprise-helm-values \
-     --set ingressController.installCRDs=false
+    -f values.yaml \
+    --set ingressController.installCRDs=false
+```
+
+#### Example values.yaml
```
Comment (Bradamant3) [inline app/kubernetes-ingress-controller/1.1.x/deployment/k4k8s-enterprise.md:144]: Can we take this up to an H3?

**Review by Bradamant3 — APPROVED** [review]
LGTM. Left one (two instances of) comment about heading levels, but optional. You can assign back to me if needed if you decide to make the change and need another approval (still learning Kong GH process ...)

**Comment by Bradamant3** [comment]
@hbagdi Lena suggested I tag you on this PR to check a few things (lgtm from a docs perspective):

- the docker image. Not sure this is quite right.
- using `konghq.com` as a placeholder in examples. More common is mockbin or httpbin ...
- 
Thanks!

File: app/kubernetes-ingress-controller/1.0.x/deployment/k4k8s-enterprise.md [hunk 2]
```diff
@@ -137,13 +144,28 @@ $ helm repo update
...
+    --set ingressController.installCRDs=false
+```
+
+### Example values.yaml
+```
+image:
+  repository: kong-docker-kong-enterprise-k8s.bintray.io/kong-enterprise-k8s
+  tag: 1.3.0.0-alpine
+  pullSecrets:
+  - kong-enterprise-k8s-docker
```
Comment (rainest) [inline app/kubernetes-ingress-controller/1.0.x/deployment/k4k8s-enterprise.md:162]: Retroactive updates to old docs are confusing. Between the new 2.3 registry and Bintray's impending demise, not sure what'd be best here. This works and lines up with the rest of the doc (the pull secret used here also changed; its name no longer matched what we created earlier) for the purpose of this PR, while recognizing that we have a larger revamp planned in DOCS-1605:

```suggestion
  repository: kong-docker-kong-enterprise-edition-docker.bintray.io/kong-enterprise-edition
  tag: 2.2.1.0-alpine
  pullSecrets:
  - kong-enterprise-edition-docker
```

File: app/kubernetes-ingress-controller/1.1.x/deployment/k4k8s-enterprise.md [hunk 3]
```diff
@@ -137,8 +144,23 @@ $ helm repo update
...
+    --set ingressController.installCRDs=false
+```
+
+### Example values.yaml
+```
+image:
+  repository: kong-docker-kong-enterprise-k8s.bintray.io/kong-enterprise-k8s
+  tag: 1.3.0.0-alpine
+  pullSecrets:
+  - kong-enterprise-k8s-docker
```
Comment (rainest) [inline app/kubernetes-ingress-controller/1.1.x/deployment/k4k8s-enterprise.md:157]:
```suggestion
  repository: kong-docker-kong-enterprise-edition-docker.bintray.io/kong-enterprise-edition
  tag: 2.2.1.0-alpine
  pullSecrets:
  - kong-enterprise-edition-docker
```

File: app/kubernetes-ingress-controller/1.0.x/deployment/k4k8s-enterprise.md (hunk 2 above)
Comment (charlie-haley) [reply app/kubernetes-ingress-controller/1.0.x/deployment/k4k8s-enterprise.md:162]: I did pull these values directly  from the URL that was previously mentioned in the docs, happy to update these 😄

**Pull request merged**